bindings/matrix-sdk-ffi/src/room_member.rs

```diff
@@ -154,3 +154,16 @@ pub fn suggested_role_for_power_level(power_level: i64) -> RoomMemberRole {
     // It's not possible to expose the constructor on the Enum through Uniffi ☹️
     RoomMemberRole::suggested_role_for_power_level(power_level)
 }
+
+#[derive(uniffi::Record)]
+pub struct ExportedRoomMember {
```

Comment:
Could we replace `RoomMember` with this `ExportedRoomMember`, or merge them one way or another?

Reply:
Thanks for merging them; can you rename `ExportedRoomMember` to `RoomMember`, then?